### PR TITLE
Add an unload script to download the active dashboards and save them

### LINKED ISSUE #97: Grafana dump script
Write the opposite of load.sh, which will use the Grafana API to dump the current dashboards so that they can be checked into Git.

### PR DESCRIPTION
**Pull request by asalkeld**
There are a couple of fields that are different:
- folderId
- overwrite
- id

closes #97

### CODE REVIEW

File: sre/openshift-sre-grafana/dashboards/alert-manager-monitoring.json [hunk 1]
```diff
@@ -2,27 +2,27 @@
   "dashboard": {
     "__requires": [
       {
-        "type": "grafana",
         "id": "grafana",
         "name": "Grafana",
+        "type": "grafana",
```
Comment (kwoodson) [inline sre/openshift-sre-grafana/dashboards/alert-manager-monitoring.json]: are these autogenerated?  Seems like the diff is just moving them down a few lines.

Comment (asalkeld) [reply sre/openshift-sre-grafana/dashboards/alert-manager-monitoring.json]: yes, it the result of running the unload script. It has json.dumps(dash_full, indent=2, sort_keys=True)
I thought the sort_keys=True would help in the long run.

**Comment by mjudeikis** [comment]
@asalkeld close with a change of priorities?

**Comment by asalkeld** [comment]
@mjudeikis yeah, there wasn't much enthusiasm for this

**Pull request closed without merging**